cpp/csv_utils_module/algorithm/csv_utils.cpp

```diff
@@ -0,0 +1,54 @@
+#include "csv_utils.hpp"
+
+namespace CsvUtils {
+
+void CreateCsvFile(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory) {
+  mgp::memory = memory;
```

Comment:
Im not tagged as reviewer, but was "around", you dont need mgp::memory = memory;, that will be deprecated soon, DispatcherGuard is enough